### ISSUE TITLE
Text input fields stop working

### ISSUE THREAD

**Issue by yellow1912** [Simonwep/selection#42]
#### Do you want to request a **feature** or report a **bug**? 
This seems to be a possible bug?
#### What is the current behavior?
After enabling the selection plugin, I can longer click on an text input to edit anything
#### If the current behavior is a bug, please provide the steps to reproduce.
Just enabling the selection plugin causes the above error
#### What is the expected behavior?
Selection should not affect other behaviors
#### Your environment:
```
Selection-version: latest version as the time this question is posted
Browser-version:  latest version as the time this question is posted (Chrome)
Operating-system:  Windows 10
```

**Comment by yellow1912** [comment]
This is related to https://github.com/Simonwep/selection/issues/41 I believe.

**Comment by Simonwep** [comment]
Whoops, I don't know why but on every event preventDefault gets called [here](https://github.com/Simonwep/selection/blob/master/src/selection.js#L206). That's a bug as far as I can tell, but there must be a reason why...

**Comment by Simonwep** [comment]
Oh yeah, on mobile everything get's really slow and swiping down causes a refresh of the page... I will take a look at it.

**Comment by Simonwep** [comment]
Fixed in c722fd310cbd1f1063e411ec3d3dfc8f32dd72ad :) Than's for reporting it!
